pkg/services/serviceaccounts/api/token.go

```diff
@@ -126,7 +126,7 @@ func (api *ServiceAccountsAPI) ListTokens(ctx *contextmodel.ReqContext) response
 // 500: internalServerError
 func (api *ServiceAccountsAPI) CurrentServiceAccount(ctx *contextmodel.ReqContext) response.Response {
 	if !ctx.IsServiceAccount {
-		return response.Error(http.StatusBadRequest, "Auth method is not service account token", errors.New("failed to get service account info"))
+		return response.JSON(http.StatusOK, nil)
```

Comment:
Not sure if it's good idea

Reply:
Any suggestions? Like reserve -1 ID for SA? In case it is not service account? Because in case of returning error it is logged in Grafana log.

Reply:
I think it's ok to log it in grafana log as API key is deprecated.

Reply:
Reverted and right now it will be error in logs again. Changes related to this change in PMM PR: https://github.com/percona/pmm/pull/3420/commits/e519d45d228e974676df8991178dbe9de209f591. Ticket also adjusted.